Tasks/Common/AzureHelpers/PrivateFunctions.ps1

```diff
@@ -58,19 +58,13 @@ function Initialize-AzureSubscription {
         $store.Add($certificate)
         $store.Close()
         $additional = @{ }
-        if ($script:isClassic -and $script:classicVersion -lt ([version]'0.8.8')) {
```

Comment:
Ting identified in the last PR this is below the minimum supported version. So no need to have this logic at all.